interfaces/IBF-dashboard/src/global.scss

```diff
@@ -57,15 +57,18 @@
 }
 
 // trigger alert colors
-
 .trigger-alert {
   .ibf-dashboard-left-column {
     .component-title {
       background-color: var(--ion-color-ibf-salmon) !important;
 
-      .component-title-item {
+      .component-title-item,
+      .today-button {
+        color: var(--ion-color-ibf-black) !important;
+
         .component-title-thumbnail {
           display: block !important;
+          background-image: url('assets/icons/alert-black.svg') !important;
```

Comment:
@jannisvisser this url is different in dashboard.page.scss, i.e.) '../../../' is missing here. Is this difference intended as a test?

Reply:
no, because the respective css files are in different places, this was actually how it was needed to work